[2022-02-15 03:25:38,232] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.mart_orders manual__2022-02-15T03:24:14.594120+00:00 [queued]>
[2022-02-15 03:25:38,256] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.mart_orders manual__2022-02-15T03:24:14.594120+00:00 [queued]>
[2022-02-15 03:25:38,257] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:25:38,260] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:25:38,261] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:25:38,278] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.mart_orders> on 2022-02-15 03:24:14.594120+00:00
[2022-02-15 03:25:38,285] {standard_task_runner.py:52} INFO - Started process 2273 to run task
[2022-02-15 03:25:38,290] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.mart_orders', 'manual__2022-02-15T03:24:14.594120+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmp96awqt98', '--error-file', '/tmp/tmp0hfqolq2']
[2022-02-15 03:25:38,292] {standard_task_runner.py:77} INFO - Job 137: Subtask model.my_new_project.mart_orders
[2022-02-15 03:25:38,425] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.mart_orders manual__2022-02-15T03:24:14.594120+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:25:38,535] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.mart_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:24:14.594120+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:24:14.594120+00:00
[2022-02-15 03:25:38,538] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:25:38,550] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models mart_orders             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:25:38,566] {subprocess.py:85} INFO - Output:
[2022-02-15 03:25:44,073] {subprocess.py:89} INFO - 03:25:44  Running with dbt=1.0.1
[2022-02-15 03:25:44,504] {subprocess.py:89} INFO - 03:25:44  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:25:44,505] {subprocess.py:89} INFO - 03:25:44
[2022-02-15 03:25:48,453] {subprocess.py:89} INFO - 03:25:48  Concurrency: 4 threads (target='dev')
[2022-02-15 03:25:48,456] {subprocess.py:89} INFO - 03:25:48
[2022-02-15 03:25:48,471] {subprocess.py:89} INFO - 03:25:48  1 of 1 START table model dbt_bytecode_training.mart_orders...................... [RUN]
[2022-02-15 03:25:51,692] {subprocess.py:89} INFO - 03:25:51  1 of 1 OK created table model dbt_bytecode_training.mart_orders................. [SUCCESS 1 in 3.22s]
[2022-02-15 03:25:51,704] {subprocess.py:89} INFO - 03:25:51
[2022-02-15 03:25:51,706] {subprocess.py:89} INFO - 03:25:51  Finished running 1 table model in 7.20s.
[2022-02-15 03:25:51,712] {subprocess.py:89} INFO - 03:25:51
[2022-02-15 03:25:51,714] {subprocess.py:89} INFO - 03:25:51  Completed successfully
[2022-02-15 03:25:51,716] {subprocess.py:89} INFO - 03:25:51
[2022-02-15 03:25:51,718] {subprocess.py:89} INFO - 03:25:51  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:25:52,134] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:25:52,163] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.mart_orders, execution_date=20220215T032414, start_date=20220215T032538, end_date=20220215T032552
[2022-02-15 03:25:52,221] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:25:52,253] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
